exec/java-exec/src/main/java/org/apache/drill/exec/opt/BasicOptimizer.java

```diff
@@ -241,7 +233,7 @@ public PhysicalOperator visitFilter(final Filter filter, final Object obj) throw
     }
 
     @Override
-    public PhysicalOperator visitUnnest(final Unnest unnest, final Object obj) throws OptimizerException {
+    public PhysicalOperator visitUnnest(final Unnest unnest, final Object obj) {
       return new org.apache.drill.exec.physical.config.UnnestPOP(null, unnest.getColumn());
```

Comment:
add `org.apache.drill.exec.physical.config.` to import statements

Reply:
Done.